src/codeenginestudio/huong/Exercise1.java

```diff
@@ -0,0 +1,47 @@
+package codeenginestudio.huong;
+
+import java.util.Arrays;
+
+public class Exercise1 {
+    public static void main(String[] args) {
+
+        String[] strArray = {"is2", "pa6" ,"Thi1s", "T4est", "3a", "th5"};
+        sortString(strArray);
+    }
+
+    public static boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch(NumberFormatException e){
+            return false;
+        }
+    }
+
+    public static char[] convertToArray (String str){
+        char[] charArray = str.toCharArray();
+        return charArray;
+    }
+
+    public static void sortString(String[] strArray){
+
+        String temp = "";
+        int max = -1;
+        for (int i = 0; i < strArray.length; i++){
+            char[] itemToArray = convertToArray(strArray[i]);
+            for(int j = 0; j <= itemToArray.length -1; j++){
+                if(isNumeric(String.valueOf(itemToArray[j])) == true){
```

Comment:
if condition valid with 2 returns are true and false. So we don't need to compare it with true here.